components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/constants/RESTServiceConstants.java

```diff
@@ -37,18 +37,22 @@
     public static final String API_GET_BY_NAME_VERSION_URI =
             "apis?query=name:" + GatewayCliConstants.API_NAME_PLACEHOLDER + "%20version:"
                     + GatewayCliConstants.VERSION_PLACEHOLDER + "%20status:PUBLISHED&expand=true";
-    public static final String PUB_RESOURCE_PATH = "/api/am/publisher/v0.14";
-    public static final String PUB_CLIENT_CERT_PATH = "/api/am/publisher/v0.14/clientCertificates";
-    public static final String ADMIN_RESOURCE_PATH = "/api/am/admin/v0.14";
-    public static final String DCR_RESOURCE_PATH = "/client-registration/v0.14/register";
-    public static final String TOKEN_PATH = "/oauth2/token";
 
-    public static final String DEFAULT_HOST = "https://localhost:9443/";
+    public static final String CONFIG_REST_VERSION = "v0.14";
+    public static final String CONFIG_PUBLISHER_ENDPOINT = "{baseURL}/api/am/publisher/{restVersion}";
+    public static final String CONFIG_ADMIN_ENDPOINT = "{baseURL}/api/am/admin/{restVersion}";
+    public static final String CONFIG_REGISTRATION_ENDPOINT = "{baseURL}/client-registration/{restVersion}/register";
+    public static final String CONFIG_TOKEN_ENDPOINT = "{baseURL}/oauth2/token";
+    public static final String REST_VERSION_TAG="{restVersion}";
+    public static final String BASE_URL_TAG="{baseURL}";
+
+    public static final String DEFAULT_HOST = "https://localhost:9443";
     public static final String DEFAULT_TRUSTSTORE_PATH = "lib/platform/bre/security/ballerinaTruststore.p12";
     public static final String DEFAULT_TRUSTSTORE_PASS = "ballerina";
     public static final String DEFAULT_KEYSTORE_PATH = "lib/platform/bre/security/ballerinaKeystore.p12";
     public static final String DEFAULT_KEYSTORE_PASS = "ballerina";
 
     public static final String CERTIFICATE_ALIAS = "Alias";
     public static final String CERTIFICATE_TIER = "Tier";
+
```

Comment:
shall we remove this blank line

Reply:
Fixed in https://github.com/wso2/product-microgateway/pull/332/commits/7dec453bb0eb0d526d5b95004e51f7b012efb678